spread-tests/main/mount-ns-layout/process.py

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env python3
+import sys
+import json
+import re
+
+_boring_fs = set(['cgroup', 'fusectl', 'debugfs', 'pstore', 'securityfs', 'mqueue', 'hugetlbfs'])
```

Comment:
Why are we ignoring these filesystems?

At least cgroup, debugfs, and securityfs are very security sensitive. At least a comment on the definition of a boring filesystem would be helpful.